docs/src/changelog.md

```diff
@@ -7,6 +7,25 @@ CurrentModule = JuMP
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## Version 1.23.4 (November 8, 2024)
+
+### Fixed
+
+ - Fixed `UnsupportedNonlinearOperator` error for the single argument
+   `LinearAlgebra.norm` (#3864)
+ - Fixed printing `MOI.Interval` with `MIME"text/latex"` (#3866)
+
+### Other
+
+ - Various minor improvements to the documentation (#3855) (#3860)
+ - Added [MathOptAI.jl](@ref) and [MathOptSymbolicAD.jl](@ref) (#3858)
+ - Clarified [`add_to_expression!`](@ref) can add two expressions (#3859)
+ - Added `SHOT` to the installation table (#3853)
+ - Improvements to test coverage (#3867) (#3868) (#3869) (#3870) (#3871) (#3872)
+   (#3873) (#3874) (#3875)
+ - JuMP now uses [`MOI.add_constrained_variable`](@ref) when adding a scalar
+   variable with bounds (#3863) (#3865)
```

Comment:
This seemed like a `MINOR` feature change, but it doesn't actually change anything user-facing, so I've bumped only the `PATCH` number.